bin/pycbc_multi_inspiral

```diff
@@ -602,59 +615,52 @@ with ctx:
                             # Calculate the power and autochi2 values for the coinc
                         # indexes (this uses the snr timeseries before the time
                         # delay, so we need to undo it. Same for normalisation)
+                        # Need all out vals to be the same length.
+                        # This means the entries that are single values
+                        # need to be repeated once per event. 
+                        num_events = None
+                        # Disctionaries for the degrees of freedom and chisqs
+                        # these are used to compute the network chisqs
```

Comment:
We compute the single detector bank and autochisquares at the moment, not the network ones.